plugins/@grouparoo/marketo/src/lib/export/destinationMappingOptions.ts

```diff
@@ -38,19 +38,19 @@ const mapTypesFromMarketoToGrouparoo = (marketoType) => {
     integer: "integer",
     boolean: "boolean",
     email: "email",
+    phone: "phoneNumber",
     float: "float",
     datetime: "date",
+    date: "string",
     score: "integer",
     currency: "float",
+    url: "url",
+    percent: "integer",
```

Comment:
This one's interesting - I would have suggested a `float`, so then `0.6` could be sent too.  Does Marketo want whole numbers or decimals?

Reply:
only decimals as integers, look: "A percentage expressed as an integer". In other words, only 2 decimals places (0.01 = 1)